app/models/edition.rb

```diff
@@ -1,6 +1,11 @@
 class Edition < ActiveRecord::Base
   has_many :weeks
   has_many :final_surveys
+  has_many :projects
   has_many :mentor_applications
   has_many :mentee_applications
+
+  def self.current
```

Comment:
hmm why not simply call Edition.last?

Reply:
Nothing specific. Wanted to make it more readable and limit the actual db query logic to model.